### ISSUE TITLE
Adapt tuner instantiation for BTB keyword processing

### ISSUE THREAD

**Issue by micahjsmith** [HDI-Project/ATM#122]
Quickstart fails with this error (reported by @csala):
```
  File "/home/xals/Projects/Pythia/MIT/ATM/atm/worker.py", line 379, in run_classifier
    params = self.tune_hyperparameters(hyperpartition)
  File "/home/xals/Projects/Pythia/MIT/ATM/atm/worker.py", line 189, in tune_hyperparameters
    r_minimum=self.datarun.r_minimum)
TypeError: __init__() got an unexpected keyword argument 'r_minimum'
```

The worker's `self.Tuner` is an instance of `btb.tuning.base.BaseTuner`, but the different subclasses take different arguments.

Implement a `atm.utilities.make_tuner` adapter that passes only the relevant kwargs.

See also:
- https://github.com/HDI-Project/ATM/blob/master/atm/utilities.py#L133
- #114 (via #112)

**Comment by beevabeeva** [comment]
Hi there. I'm posting here in case its the same issue:
After executing ` python scripts/worker.py` I get the following output, repeatedly till killing the process :

~~~~
Computing on datarun 1
Selector: <class 'btb.selection.uniform.Uniform'>
Tuner: <class 'btb.tuning.uniform.Uniform'>
Error choosing hyperparameters: datarun=<ID = 1, dataset ID = 1, strategy = uniform__uniform, budget = classifier (100), status: running>
Traceback (most recent call last):
  File "/home/research/ATM2/atm/atm-env/lib/python2.7/site-packages/atm-0.1.1.dev0-py2.7.egg/atm/worker.py", line 379, in run_classifier
    params = self.tune_hyperparameters(hyperpartition)
  File "/home/research/ATM2/atm/atm-env/lib/python2.7/site-packages/atm-0.1.1.dev0-py2.7.egg/atm/worker.py", line 189, in tune_hyperparameters
    r_minimum=self.datarun.r_minimum)
TypeError: __init__() got an unexpected keyword argument 'r_minimum'

Something went wrong. Sleeping 1 seconds.
~~~~

I tried changing to the stable git branch, but I get the same result.

Thanks.

**Comment by csala** [comment]
> Hi there. I'm posting here in case its the same issue:

It is, indeed, the same error.

This should be fixed once the PR #124 is merged.

Regarding `stable` branch, make sure to have switched to it before installing everything!
If you checkout master and install from there and then switch to stable you will still have the dependencies from master installed and it will fail anyway.

If you checkout to stable and then create a fresh virtualenv and install, this error should go away.

**Comment by beevabeeva** [comment]
Thanks so much, again @csala !
